projects/transformers/run.py

```diff
@@ -106,8 +108,10 @@ def main():
 
         # Initialize wandb now to include the logs that follow.
         # For now, only support early wandb logging when running one experiment.
+        distributed_initialized = torch.distributed.is_initialized()
+        rank = -1 if not distributed_initialized else torch.distributed.get_rank()
```

Comment:
`rank` is only used in the early_init function call - in this case, shouldn't these two lines be inside the if statement?

Reply:
Fair. I'll fix this in a soon-to-come PR.